relayer/relayer.go

```diff
@@ -154,32 +190,29 @@ func (r *Relayer) RelayMessage(warpLogInfo *vmtypes.WarpLogInfo, metrics *Messag
 		return err
 	}
 
+	r.logger.Info(
+		"Unpacked warp message",
+		zap.String("chainID", r.sourceChainID.String()),
+		zap.String("warpMessageID", warpMessageInfo.WarpUnsignedMessage.ID().String()),
+	)
+
 	// Check that the warp message is from a support message protocol contract address.
 	messageManager, supportedMessageProtocol := r.messageManagers[warpLogInfo.SourceAddress]
 	if !supportedMessageProtocol {
 		// Do not return an error here because it is expected for there to be messages from other contracts
 		// than just the ones supported by a single relayer instance.
 		r.logger.Debug(
-			"Warp message from unsupported message protocol address. not relaying.",
+			"Warp message from unsupported message protocol address. Not relaying.",
 			zap.String("protocolAddress", warpLogInfo.SourceAddress.Hex()),
 		)
 		return nil
 	}
 
-	requestID := r.getAndIncrementRequestID()
-
-	// Add an element to the inbound messages map, keyed by the requestID
-	// This allows RouteToMessageChannel to forward inbound messages to the correct channel
-	// Note: It's technically possible to block indefinitely here if the inbound handler is processing an flood of non-relevant app responses. This would occur if
-	// the RLocks are acquired more quickly than they are released in RouteToMessageChannel. However, given that inbound messages are rate limited and that the
-	// RLock is only held for a single map access, this is likely not something we need to worry about in practice.
-	messageResponseChan := make(chan message.InboundMessage, peers.InboundMessageChannelSize)
-	r.messageChanLock.Lock()
-	r.messageChanMap[requestID] = messageResponseChan
-	r.messageChanLock.Unlock()
+	// Increment the request ID to use for this message
+	r.currentRequestID++
```

Comment:
It looks like previously we used the current value before incrementing, but now we will increment first and then use the value. I would assume this is inconsequential, but just want to call it out to confirm.

Reply:
Good catch. It shouldn't matter, since the request ID is only used by the relayer in memory, but I've moved the increment to after it's used to remain consistent.